build.sbt

```diff
@@ -20,7 +20,7 @@ developers := List(
 )
 
 scalaVersion := "3.4.2"
-crossScalaVersions := Seq("2.12.18", "2.12.19", "2.13.13", "2.13.14", "3.3.3", "3.4.2")
+crossScalaVersions := Seq("2.12.18", "2.12.20", "2.13.13", "2.13.14", "3.3.3", "3.4.2")
```

Comment:
@saeltz , are there any plans to merge PRs on a scheduled basis or does that happen impromptu? I am asking, as I plan to upgrade a project where I work, with scala 2.12.20 and it would super helpful to have a timeline (if I can) for the upgrade.

Reply:
Raised #869 to bundle 2.12.20/3.5.0 update